ginga/gtk3w/GtkHelp.py

```diff
@@ -744,6 +747,258 @@ def close_page(self, subwin):
         self._update_area_size()
 
 
+class Splitter(Gtk.Layout):
+    """
+    Splitter type widget for Gtk.
+    """
+    def __init__(self, orientation='horizontal'):
+        Gtk.Layout.__init__(self)
+
+        self.orientation = orientation
+        self._sizes = []
+        self._dims = (0, 0)
+        self.children = []
+        self.thumbs = []
+        self.kbdmouse_mask = 0
+
+        mask = self.get_events()
+        self.set_events(mask |
+                        Gdk.EventMask.ENTER_NOTIFY_MASK |
+                        Gdk.EventMask.LEAVE_NOTIFY_MASK
+                        )
+
+        self.connect("size-allocate", self._size_allocate_cb)
+        modify_bg(self, "gray50")
+
+    def add_widget(self, widget):
+        rect = self.get_allocation()
+        wd, ht = rect.width, rect.height
+
+        self.children.append(widget)
+
+        if len(self.children) == 1:
+            widget.set_size_request(wd, ht)
+            self.put(widget, 0, 0)
+            sizes = self._sizes
+            if len(sizes) == 0:
+                pos = wd if self.orientation == 'horizontal' else ht
+                sizes = [pos]
+            self.set_sizes(sizes)
+
+        else:
+            if self.orientation == 'horizontal':
+                thumbfile, _w, _h = 'vdots.png', 8, 26
+            else:
+                thumbfile, _w, _h = 'hdots.png', 26, 8
+            iconfile = os.path.join(icondir, thumbfile)
```

Comment:
@pllim, I think `get_pkg_data_filename()` does a search of all the package directories, right?  I think in this case, all the icons are in the `icons` directory, and we know exactly where that is.  I think it is actually clearer the way it is written, and not necessary to do a search.

Perhaps when (if) we have more example data files installed (in several different folders) it would make sense to me to use `get_pkg_data_filename()`.